client/client.go

```diff
@@ -0,0 +1,36 @@
+package client
+
+import (
+	"context"
+
+	sdkparams "github.com/cosmos/cosmos-sdk/simapp/params"
+	"github.com/umee-network/umee/v4/sdkclient"
+)
+
+type Client struct {
+	sdkclient.Client
+}
+
+// Initializes a cosmos sdk client context and transaction factory for
+// signing and broadcasting transactions
```

Comment:
You could add a little something about how this Client inherits the queries and transactions of a normal Cosmos SDK chain with additional tx/queries specific to the Umee chain.

Reply:
OK